x11-misc/lightdm-gtk-greeter/lightdm-gtk-greeter-2.0.9.ebuild

```diff
@@ -16,7 +16,7 @@ SLOT="0"
 KEYWORDS="~alpha amd64 arm arm64 ~loong ppc ppc64 ~riscv x86"
 IUSE="appindicator branding"
 
-DEPEND="x11-libs/gtk+:3
+DEPEND="x11-libs/gtk+:3[X]
```

Comment:
This is affecting RDEPEND though, so it must be a revbump.

Also, seeing BDEPEND split the DEPEND and RDEPEND blocks is really odd and may have contributed to you not easily noticing the above.